backend/tests/integration/orcid/cassettes/TestOrcidPusherPutUpdatedWork.test_push_updated_work_invalid_data_putcode.yaml

```diff
@@ -53,118 +55,6 @@ interactions:
-      \    author = \"Rossoni, A.\",\n    doi = \"10.9999/test.orcid.push\",\n   \
-      \ month = \"5\",\n    title = \"{Some Results Arising from the Study of ORCID\
-      \ push}\",\n    year = \"2017\"\n}\n</work:citation-value></work:citation><work:type>journal-article</work:type><common:external-ids><common:external-id><common:external-id-type>other-id</common:external-id-type><common:external-id-value>999</common:external-id-value><common:external-id-url>http://inspirehep.net/record/999</common:external-id-url><common:external-id-relationship>self</common:external-id-relationship></common:external-id><common:external-id><common:external-id-type>doi</common:external-id-type><common:external-id-value>10.9999/test.orcid.push</common:external-id-value><common:external-id-url>http://dx.doi.org/10.9999/test.orcid.push</common:external-id-url><common:external-id-relationship>self</common:external-id-relationship></common:external-id></common:external-ids><work:url>http://inspirehep.net/record/999</work:url><work:contributors><work:contributor><work:credit-name>Rossoni,\
-      \ A.</work:credit-name><work:contributor-attributes><work:contributor-sequence>first</work:contributor-sequence><work:contributor-role>author</work:contributor-role></work:contributor-attributes></work:contributor></work:contributors></work:work>"
-    headers:
-      Accept:
-      - application/orcid+json
-      Accept-Encoding:
-      - gzip, deflate
-      Connection:
-      - keep-alive
-      Content-Length:
-      - '1588'
-      Content-Type:
-      - application/orcid+xml
-    method: POST
-    uri: https://api.orcid.org/v2.0/0000-0003-1134-6827/work
-  response:
-    body:
-      string: '{"response-code":409,"developer-message":"409 Conflict: You have already
-        added this activity (matched by external identifiers), please see element
-        with put-code 71156708. If you are trying to edit the item, please use PUT
-        instead of POST.","user-message":"There was an error when updating the record.
-        Please try again. If the error persists, please contact INSPIRE-HEP for assistance.","error-code":9021,"more-info":"https://members.orcid.org/api/resources/troubleshooting"}'
-    headers:
-      Access-Control-Allow-Origin:
-      - '*'
-      Cache-Control:
-      - no-cache, no-store, max-age=0, must-revalidate
-      Connection:
-      - keep-alive
-      Content-Length:
-      - '474'
-      Content-Type:
-      - application/orcid+json; qs=2;charset=UTF-8
-      Date:
-      - Fri, 27 Mar 2020 11:22:09 GMT
-      Expires:
-      - '0'
-      Pragma:
-      - no-cache
-      Server:
-      - nginx/1.16.1
-      X-Content-Type-Options:
-      - nosniff
-      X-Frame-Options:
-      - DENY
-      X-XSS-Protection:
-      - 1; mode=block
-    status:
-      code: 409
-      message: Conflict
```

Comment:
what happened here?

Reply:
To me the new one looks correctly:

https://github.com/inspirehep/inspirehep/blob/6ce3689b5c64f02bec3aa91ff84a70f77a1a2795/backend/inspirehep/orcid/domain_models.py#L160-L173

There is a sequence:
- _post_or_put_work() (PUT) --> PutcodeNotFoundPutException 404
- _cache_all_author_putcodes() (GET) --> 200
- _post_or_put_work() (PUT) --> 201.